server/routes/post.routes.js

```diff
@@ -0,0 +1,15 @@
+const validateToken = require('../middlewares/validateToken.js')
+
+module.exports = app => {
+    const posts = require('../controllers/post.controller.js')
+
+    var router = require('express').Router()
+
+    router.post('/', validateToken, posts.insertPosts)
+    router.get('/', validateToken, posts.getUserPosts)
+    router.get('/all', posts.getAllPosts)

```

Comment:
Please take a look at RESTful API structure on Google, a bit confusing URLs...

Reply:
could you give me a hint?